Healthy/Classes/Modules/Splash/SplashViewController.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+  final class SplashViewController: UIViewController {
+
+        // MARK: Outlets
+
+      @IBOutlet private weak var startCookingButton: UIButton!
+      @IBOutlet private weak var logoCaptionLabel: UILabel!
+      @IBOutlet private weak var headerTitleLabel: UILabel!
+      @IBOutlet private weak var headerCaptionLabel: UILabel!
```

Comment:
Change the Outlets from `private` to  `private(set)`, so that we can test them when needed.